frontend/src/lib/instance-views/elements/VStack.svelte

```diff
@@ -27,13 +26,15 @@
 	<Error type="Incorrect Data" message={errorMessage} />
 {:else}
 	<div class="flex flex-col">
-		{#each keys as key}
-			{#if spec.keys && spec.keys[key] !== undefined && spec.keys[key].type !== undefined}
+		{#each Object.keys(spec.keys) as key}
+			{#if key in jsonData}
 				<svelte:component
 					this={resolveElementType(key)}
 					spec={spec.keys[key]}
 					data={typeof jsonData[key] === 'object' ? JSON.stringify(jsonData[key]) : jsonData[key]}
 				/>
+			{:else}
+				<p class="text-error">ERROR: no data for key <b>{key}</b></p>
```

Comment:
Don't think we should show an error here. This can be intentional and useful. You could have a key that's just sometimes there or for one type of element you have one key for the other type you have another key.